scripts/provision/roles/web/tasks/mapstory.yml

```diff
@@ -67,13 +68,24 @@
   tags: [settings]
 
 - name: write secret key
-  template: src=roles/web/files/local_key.py dest={{ project_path }}/{{ project_name }}/settings/local_key.py
+  template: src=roles/web/files/local_key.py dest=/tmp/local_key.py
   sudo_user: '{{ application_user }}'
   tags: [settings]
+#  notify: restart django
+
+- name: mv project local settings across synced file system if necessary
+  command: mv /tmp/local_key.py {{ project_path }}/{{ project_name }}/settings/local_key.py
+  sudo_user: '{{ application_user }}'
   notify: restart django
+  tags: [settings]
+ 
+- name: write project local settings - sync filesystem safe
+  template: src=roles/web/files/local_settings.py.j2 dest=/tmp/local_settings.py
+  sudo_user: '{{ application_user }}'
+  tags: [settings]
 
-- name: write project local settings
-  template: src=roles/web/files/local_settings.py.j2 dest={{ project_path }}/{{ project_name }}/settings/local_settings.py
+- name: mv project local settings across synced file system if necessary
```

Comment:
during provisioning it avoids a known limitation of django manage to write to a synced filesystem. Just puts output somewhere tmp then moves it to the desired target.